app/src/main/res/navigation/nav_graph.xml

```diff
@@ -62,6 +68,11 @@
             android:name="session_id"
             app:argType="string"
             app:nullable="true" />
+        <argument
+            android:name="initializeSearchShortcuts"
```

Comment:
naming nit: instead of `initializeSearchShortcuts` can we do something like `displaySearchShortcuts`? I'm worried that saying initialize implies a search is being triggered.